sgrep.yml

```diff
@@ -0,0 +1,26 @@
+rules:
```

Comment:
Also, can you add a link to the documentation for this format in a comment at the top of the file?